src/openrct2/interface/Window.h

```diff
@@ -137,6 +137,12 @@ struct sprite_focus {
     uint8_t zoom; //0x489
 };
 
+union viewport_focus
+{
```

Comment:
I'm not convinced that this is any better to the previous this all gets stuffed into the var_480 union of the window. This is just a subset of that union.